docs/UserGuide.md

```diff
@@ -11,18 +12,18 @@
    > Having any Java 8 version is not enough. <br>
    This app will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `dearjim.jar` from the [releases](../../../releases) tab.
```

Comment:
the GUI mockup shown here is quite a deviation from the addressbook-level4 GUI, and looks quite basic; while this is acceptable, remember that you can reuse the GUI code from AB-L4 for much of the things.
it is ok to remove the browser component totally, but do think of making use of the available space effectively to display useful information to the user.